Assets/ZSSRichTextEditor.js

```diff
@@ -107,7 +107,12 @@ ZSSEditor.init = function(callbacker, logger) {
         // Ensure we only insert plaintext from the pasteboard
         e.preventDefault();
         var plainText = (e.originalEvent || e).clipboardData.getData('text/plain');
-        document.execCommand('insertText', false, plainText);
+        if (plainText.length > 0) {
```

Comment:
If it has both, the text will paste and the image will not. IMO the best way to solve this is to have the native code trigger with all pastes, where it can handle multiple items on the pasteboard.

Not sure if that's something to implement now, or as a second phase after this one?